src/components/Layout/Header.tsx

```diff
@@ -24,11 +27,109 @@ const onRightClickLogo = (event: React.MouseEvent): void => {
     }
 }
 
+const navLinks = [
+    {
+        section: 'Product',
+        items: [
+            {
+                title: 'Code Search',
+                href: '/code-search',
+            },
+            {
+                title: 'Batch Changes',
+                href: '/batch-changes',
+            },
+            {
+                title: 'Code Insights',
+                href: '/code-insights',
+            },
+            {
+                title: 'Code Intelligence',
+                href: 'https://docs.sourcegraph.com/code_intelligence',
+            },
+        ],
+    },
+    {
+        section: 'Resources',
+        items: [
+            {
+                title: 'Blog',
+                href: '/blog',
+            },
+            {
+                title: 'Learn',
+                href: 'https://learn.sourcegraph.com/',
+            },
+            {
+                title: 'Dev Tool Time',
+                href: 'https://www.youtube.com/playlist?list=PL6zLuuRVa1_iDEP4EicZ8972RgyccCRGF',
+            },
+            {
+                title: 'Sourcegraph Podcast',
+                href: '/podcast',
+            },
+            {
+                title: 'Case Studies',
+                href: '/case-studies',
+            },
+        ],
+    },
+    {
+        section: 'Use Cases',
+        items: [
+            {
+                title: 'All Use Cases',
+                href: '/use-cases',
```

Comment:
A link to the Use Case landing page has been added to the top of the Use Case dropdown as a result of stakeholder review. This differs from the initial spec.